charts/datadog/values.yaml

```
@@ -421,6 +421,9 @@ datadog:
       #   - send_distribution_buckets: true
       #     timeout: 5
 
+  # datadog.ignoreAutoConfig -- List of integration to ignore auto_conf.yaml.
+  ignoreAutoConfig: []
```

we can provide an example here
```suggestion
  ignoreAutoConfig: []
  #  - redisdb
  #  - kubernetes_state
```